scripts/kurtosis/network_params.json

```diff
@@ -2,7 +2,8 @@
     "mev_type": "full",
     "participants": [
         {
-            "el_client_type": "geth",
+            "el_client_type": "nethermind",
+            "el_client_image": "nethermind/nethermind:1.21.0",
```

Comment:
Let's use 1.21.1 because of some issue discovered on 1.21.0 - if we would keep neth ofc